source/base/data_out_base.cc

```diff
@@ -6220,7 +6220,8 @@ write_pvd_record (std::ostream &out,
                   const std::vector<std::pair<double,std::string> >  &times_and_names) const
 {
   AssertThrow (out, ExcIO());
-
+   
+  out.precision(12);
```

Comment:
In ASPECT, we actually did something recently: unconditionally setting the precision in which we output the time to 12 digits or so. I think it makes sense -- it's just a single number, so the amount of data we write is not of any kind of relevance, and it's a hassle to have to do so through a parameter. I'm in favor of unconditionally leaving it at 12 digits.